### PR TITLE
test: Integration test

### PR DESCRIPTION
**Pull request by vahidlazio**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -36,6 +36,7 @@ jobs:
       working-directory: example
       run: flutter pub get
 
+    - run: echo API_KEY=${{ secrets.TEST_API_KEY }} > example/.env
```
Comment (nicklasl) [inline .github/workflows/ci.yaml:39]: not needed?

File: integration_test/integration_test.dart [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:confidence_flutter_sdk/confidence_flutter_sdk.dart'; // Adjust the import to your actual plugin
+
+void main() {
+  group('Confidence', () {
+
+    });
```
Comment (andreas-karlsson) [inline integration_test/integration_test.dart:8]: Is this test doing anything?

File: .github/workflows/android-test.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        uses: subosito/flutter-action@v2
+        with:
+          flutter-version: ${{ env.FLUTTER_VERSION }}
+          channel: ${{ env.FLUTTER_CHANNEL }}
+
+      - name: Enable KVM
+        run: |
+          echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+          sudo udevadm control --reload-rules
+          sudo udevadm trigger --name-match=kvm
```
Comment (nicklasl) [inline .github/workflows/android-test.yaml:33]: can we add some comments around why this is needed. Where does it come from etc?

File: .github/workflows/ios-test.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+      - name: Remove the submodule
+        working-directory: ios/Classes
+        run: rm -rf confidence-sdk
+
+      - name: Remove git submodule
+        working-directory: ios/Classes
+        run: git rm confidence-sdk
+
+      - uses: futureware-tech/simulator-action@v3
```
Comment (nicklasl) [inline .github/workflows/ios-test.yaml:36]: This action only have 44 stars, should we look for something more well used?

File: integration_test/integration_test.dart (hunk 2 above)
Comment (vahidlazio) [reply integration_test/integration_test.dart:8]: no, deleted.

File: .github/workflows/ios-test.yaml (hunk 4 above)
Comment (vahidlazio) [reply .github/workflows/ios-test.yaml:36]: this is the most famous one I found. we can change it if it becomes flaky in the future or what is your concern?

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (vahidlazio) [reply .github/workflows/ci.yaml:39]: we need this since the example app needs this file to be provided in the assets.

**Review by nicklasl — APPROVED** [review]

**Pull request merged**